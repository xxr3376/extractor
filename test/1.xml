<?xml version="1.0"?>
<conf>
  <define>
    <regex name="find_id" value="^.*?(\d+).*?$" target="1" mode="DOTALL"/>
    <regex name="full_url" value=".*full_url=([^&amp;]+)(&amp;|$)" target="1" mode="DOTALL"/>
    <regex name="short_url" value=".*short_url=([^&amp;]+)(&amp;|$)" target="1" mode="DOTALL"/>
    <regex name="title" value=".*title=([^&amp;]+)(&amp;|$)" target="1" mode="DOTALL"/>
  </define>
  <page>
    <loop name="data" selector=".WB_cardwrap [mid]" valid="id, user_id, content, timestamp">
      <field name="id" action="attr(mid)|int"/>
      <wrap selector="a.W_texta">
        <field name="nickname" action="attr(nick-name)"/>
        <field name="user_href" action="attr(href)"/>
        <field name="user_id" action="attr(usercard) | regex(find_id) | int"/>
      </wrap>
      <field name="content" selector="p.comment_txt" action="text"/>
      <wrap selector=".feed_action_info">
        <field name="retweet_count" child="1" action="text | regex(find_id) | default(0) | int "/>
        <field name="comment_count" child="2" action="text | regex(find_id) | default(0) | int "/>
        <field name="like_count" child="3" action="text | regex(find_id) | default(0) | int "/>
      </wrap>
      <wrap selector=".feed_from">
        <wrap selector="a" order="0">
          <field name="timestamp" action="attr(date) | int"/>
          <field name="url" action="attr(href)"/>
        </wrap>
        <field name="source" selector="a" order="1" action="text"/>
      </wrap>
      <wrap selector=".media_box ul a">
        <field name="video_full_url" action="attr(action-data) | regex(full_url)|unquote"/>
        <field name="video_short_url" action="attr(action-data) | regex(short_url)"/>
        <field name="video_title" action="attr(action-data) | regex(title) | unquote | decode(utf-8)"/>
      </wrap>
      <field name="video_brief" selector=".WB_feed_spec_brieftxt2" action="text | strip"/>
    </loop>
  </page>
</conf>
