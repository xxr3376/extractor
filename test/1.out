{"data": [{"user_id": 2012397877, "video_short_url": "http://t.cn/hgtk2G", "url": "http://weibo.com/2012397877/C2LR2Au6T", "timestamp": 1423049087000, "video_title": "[\u9646\u5c0f\u51e4]\u51e4\u821e\u4e5d\u5929", "content": "\u3010\u89c6\u9891\uff1a [\u9646\u5c0f\u51e4]\u51e4\u821e\u4e5d\u5929\u3011 \uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09\u5c31\u662f\u559c\u6b22\u8fd9\u9996\u4e3b\u9898\u66f2\u800c\u5df2\uff0c\u81ea\u5df1\u4f20\u7684\u539f\u7248\u627e\u4e0d\u5230\u4e86\u5443\u2026\u2026\u6bd4\u8d77\u6c99\u66fc\u6211\u8fd8\u662f\u559c\u6b22\u859b\u51b0\u8fd9\u4e2a\u4f1a\u8138\u7ea2\u7684\u6bcd\u8001\u864e\u591a\u4e00\u70b9\u554a\uff0c\u9646\u5c0f\u9e21\u548c\u5973\u4eba\u90fd\u662fBE\u4ec0\u4e48\u7684\u592a\u5fc3\u585e\u4e86\u2190 \u2190 http://t.cn/hgtk2G ", "source": "\u4f18\u9177\u571f\u8c46", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XNDQzMTczNjA=.html", "user_href": "http://weibo.com/aizhedian", "retweet_count": 0, "nickname": "S\u672c\u82cf\u4e2d\u4e8c\u95f4\u6b47\u75c7\u5019\u7fa4\u60a3", "id": 3806606128695555}, {"user_id": 1957732864, "video_short_url": "http://t.cn/8sM5m1R", "url": "http://weibo.com/1957732864/C2LQZFCGk", "timestamp": 1423049080000, "video_brief": "\u8c28\u4ee5\u6b64\u7247\u9001\u7ed9\u6211\u7684\u65b0\u5a18\u77f3\u8bd7\uff0c\u5e76\u7eaa\u5ff5\u6211\u4eec\u7684\u5a5a\u793c~~~2014\u5e743\u670829\u65e5 ...", "video_title": "\u5a5a\u793c\u9b54\u672fMV\u300a\u4e3a\u4f60\u5199\u8bd7\u300b", "content": "\u6211\u5feb\u54ed\u4e86\uff01\u3010\u89c6\u9891\uff1a\u5a5a\u793c\u9b54\u672fMV\u300a\u4e3a\u4f60\u5199\u8bd7\u300b\u3011 \uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09 |\u5a5a\u793c\u9b54\u672fMV... ", "source": "\u4f18\u9177\u571f\u8c46", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XNjkyMzc2MTc2.html", "user_href": "http://weibo.com/u/1957732864", "retweet_count": 1, "nickname": "\u8fd9-\u7231\u591a\u7f8eMxu", "id": 3806606099920144}, {"user_id": 5031465443, "video_short_url": "http://t.cn/RZsprfq", "url": "http://weibo.com/5031465443/C2LQZ6o6f", "timestamp": 1423049077000, "video_brief": "\u7a0d\u540e\u8865\u5145\u89c6\u9891\u7b80\u4ecb", "video_title": "2015\u5965\u5df4\u9a6c\u54c8\u4f5b\u5927\u5b66\u6f14\u8bb2--\u5927\u5b66\u751f\u7684\u793c\u7269_\u6807\u6e05", "content": "2015\u5965\u5df4\u9a6c\u54c8\u4f5b\u5927\u5b66\u6f14\u8bb2--\u5927\u5b66\u751f\u7684\u793c\u7269_\u6807\u6e05 \uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09 |2015\u5965\u5df4\u9a6c... ", "source": "\u4f18\u9177\u571f\u8c46", "comment_count": 1, "like_count": 1, "video_full_url": "http://v.youku.com/v_show/id_XODc4OTUxNTM2.html", "user_href": "http://weibo.com/u/5031465443", "retweet_count": 0, "nickname": "umaykg", "id": 3806606091522611}, {"user_id": 2447929264, "video_short_url": "http://t.cn/RwvONIl", "url": "http://weibo.com/2447929264/C2LQYuJtb", "timestamp": 1423049077000, "video_brief": "2015\u8d85\u7ea7\u7897\u5e7f\u544a \u81ea\u7531\u4fa0\u7f8e\u4e3d\u65b0\u5927\u9646", "video_title": "2015\u8d85\u7ea7\u7897\u5e7f\u544a+\u81ea\u7531\u4fa0\u7f8e\u4e3d\u65b0\u5927\u9646", "content": "\u68a6\u91cc\u6211\u8d70\u904d\u4e86\u5168\u4e16\u754c  JEEP\u5e7f\u544a\u90fd\u62cd\u7684\u597d@Jeep\u4e2d\u56fd\u7ad9 \uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09 |2015\u8d85\u7ea7\u7897... ", "source": "\u4f18\u9177\u571f\u8c46", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XODg2MTE5NjQw.html", "user_href": "http://weibo.com/u/2447929264", "retweet_count": 0, "nickname": "\u81ed\u76ae\u5320c", "id": 3806606087324629}, {"user_id": 3759256232, "video_short_url": "http://t.cn/RZF06i9", "url": "http://weibo.com/3759256232/C2LQTu1Lj", "timestamp": 1423049064000, "video_brief": "\u5361\u5730\u4e9a\u53d1\u5e03\u7531\u8096\u6069\u00b7\u57c3\u5229\u65af\u6267\u5bfc\u7684\u5168\u65b0\u5f71\u7247\u300aThe Proposal\u300b\uff0c\u81f4\u656c...", "video_title": "\u5361\u5730\u4e9aCartier+\u5168\u65b0\u5f71\u7247\u300aThe+Proposal\u300b", "content": "\u3010\u89c6\u9891\uff1a \u5361\u5730\u4e9aCartier \u5168\u65b0\u5f71\u7247\u300aThe Proposal\u300b\u3011 \uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09 |\u5361\u5730\u4e9aCart... ", "source": "\u4f18\u9177\u571f\u8c46", "comment_count": 0, "like_count": 1, "video_full_url": "http://v.youku.com/v_show/id_XODcwMTEzODcy.html", "user_href": "http://weibo.com/u/3759256232", "retweet_count": 0, "nickname": "\u5996\u9b54\u5316\u7684\u4e03\u4e00", "id": 3806606037156617}, {"user_id": 1732530193, "video_short_url": "http://t.cn/RZsKYJO", "url": "http://weibo.com/1732530193/C2LQSzRrD", "timestamp": 1423049060000, "video_brief": "", "video_title": "\u7b2c17\u8bdd+\u963f\u6ce2\u7684\u6d77\u8d3c", "content": "\u6211\u5728\u770b\u3010\u89c6\u9891\uff1a\u6668\u66e6\u516c\u4e3b 17\u3011\uff1a\u585e\u5df4\u65af\u9171\u7684\u5c0f\u5f1f\u51fa\u73b0\u3002 |\u7b2c17\u8bdd \u963f... \uff08\u5206\u4eab\u81ea @\u4f18\u9177\u52a8\u6f2b\uff09", "source": "\u4f18\u9177\u571f\u8c46", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XODg1OTQyNTEy.html", "user_href": "http://weibo.com/u/1732530193", "retweet_count": 0, "nickname": "\u98de\u821e\u7684\u900f\u89c6\u706b\u82b1", "id": 3806606028546925}, {"user_id": 1799312905, "video_short_url": "http://t.cn/RwvMsDZ", "url": "http://weibo.com/1799312905/C2LQSgWbl", "timestamp": 1423049062000, "video_brief": "", "video_title": "\u5de8\u641e\u7b11\u6853\u53f0\u4eba\u62dc\u5e74", "content": "\u6853\u53f0\u4eba\u62dc\u5e74 |\u5de8\u641e\u7b11\u6853\u53f0...\uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09", "source": "\u5206\u4eab\u6309\u94ae", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XODg0ODg5MjA0.html", "user_href": "http://weibo.com/u/1799312905", "retweet_count": 1, "nickname": "CURVA_NORD-Wang-R-M", "id": 3806606024036903}, {"user_id": 3896659137, "video_short_url": "http://t.cn/Rwvrf0z", "url": "http://weibo.com/3896659137/C2LQNyeBu", "timestamp": 1423049051000, "video_brief": "\u963f\u53e4\u8339\u90a3\u4e2a\u62e5\u62b1\u5f88\u7a81\u7136\uff0c\u611f\u8c22@ultra_H\u7684\u53ca\u65f6\u62cd\u6444", "video_title": "\u3010\u4e1c\u4eac\u5965\u7279\u66fc\u6b63\u6708\u8282\u3011\u5965\u7279\u66fc\u4eec\u9000\u573a\u77ac\u95f4~", "content": "\u3010\u89c6\u9891\uff1a \u3010\u4e1c\u4eac\u5965\u7279\u66fc\u6b63\u6708\u8282\u3011\u5965\u7279\u66fc\u4eec\u9000\u573a\u77ac\u95f4~\u3011 \uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09 |\u3010\u4e1c\u4eac\u5965\u7279... ", "source": "\u4f18\u9177\u571f\u8c46", "comment_count": 1, "like_count": 2, "video_full_url": "http://v.youku.com/v_show/id_XODg2NDUzMDU2.html", "user_href": "http://weibo.com/ultramanyy", "retweet_count": 13, "nickname": "\u5965\u7279\u66fc\u5c0f\u6c60\u5858", "id": 3806605978159292}, {"user_id": 3896659137, "video_short_url": "http://t.cn/Rwvrcdm", "url": "http://weibo.com/3896659137/C2LQGcQf5", "timestamp": 1423049033000, "video_brief": "\u9ad8\u51b7\u963f\u53e4\u8339\u56e0\u4e3a\u5927\u5bb6\u53ea\u987e\u7740\u62cd\u89c6\u9891\u800c\u4e0d\u8ddf\u4ed6\u8df3\u5237\u813e\u6c14\u4e86~~\u5176\u5b9e\u4e0d\u662f\u4e0d...", "video_title": "\u3010\u4e1c\u4eac\u5965\u7279\u66fc\u6b63\u6708\u8282\u3011\u963f\u53e4\u8339\u5965\u7279\u66fc\u8df3\u97f3\u5934", "content": "\u3010\u89c6\u9891\uff1a \u3010\u4e1c\u4eac\u5965\u7279\u66fc\u6b63\u6708\u8282\u3011\u963f\u53e4\u8339\u5965\u7279\u66fc\u8df3\u97f3\u5934\u3011 \uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09 |\u3010\u4e1c\u4eac\u5965\u7279... ", "source": "\u4f18\u9177\u571f\u8c46", "comment_count": 0, "like_count": 2, "video_full_url": "http://v.youku.com/v_show/id_XODg2NDUwODE2.html", "user_href": "http://weibo.com/ultramanyy", "retweet_count": 19, "nickname": "\u5965\u7279\u66fc\u5c0f\u6c60\u5858", "id": 3806605903060759}, {"user_id": 2186128204, "video_short_url": "http://t.cn/RZmNCPg", "url": "http://weibo.com/2186128204/C2LQFjyYL", "timestamp": 1423049031000, "video_brief": "How Engines Work\nRolls-Royce is one of the world's leading ...", "video_title": "\u7f57\u5c14\u65af\u00b7\u7f57\u4f0a\u65af\u5ba3\u4f20\u7247+-+\u53d1\u52a8\u673a\u662f\u600e\u6837\u5de5\u4f5c\u7684", "content": "\u7f57\u5c14\u65af\u00b7\u7f57\u4f0a\u65af\u5ba3\u4f20\u7247 - \u53d1\u52a8\u673a\u662f\u600e\u6837\u5de5\u4f5c\u7684 |\u7f57\u5c14\u65af\u00b7\u7f57...\uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09", "source": "\u5206\u4eab\u6309\u94ae", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XODgwNzg2MTQ4.html", "user_href": "http://weibo.com/u/2186128204", "retweet_count": 0, "nickname": "sMetal", "id": 3806605894662695}, {"user_id": 1752001282, "video_short_url": "http://t.cn/Rwvrcws", "url": "http://weibo.com/1752001282/C2LQDw6nJ", "timestamp": 1423049026000, "video_brief": "\u963f\u5c14\u5351\u65af\u96ea\u5c71\u6ed1\u96ea\u6c42\u5a5a", "video_title": "\u745e\u58eblaax\u6ed1\u96ea", "content": "\u745e\u58eblaax\u6ed1\u96ea |\u745e\u58eblaax\u6ed1...\uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09", "source": "\u5206\u4eab\u6309\u94ae", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XODgzODcyNDA0.html", "user_href": "http://weibo.com/luoluozixin", "retweet_count": 0, "nickname": "jiissi", "id": 3806605877651031}, {"user_id": 1789221600, "video_short_url": "http://t.cn/RhuilJs", "url": "http://weibo.com/1789221600/C2LQBrVGi", "timestamp": 1423049021000, "video_brief": "[20130503]Ombre Cake", "video_title": "\u6761\u7eb9\u86cb\u7cd5[HowToCookThat]", "content": "\u6761\u7eb9\u86cb\u7cd5[HowToCookThat] |\u6761\u7eb9\u86cb\u7cd5[H...\uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09", "source": "\u5206\u4eab\u6309\u94ae", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XNjY2NjI1NDU2.html", "user_href": "http://weibo.com/u/1789221600", "retweet_count": 0, "nickname": "\u8d75\u4e3b\u4efb\u548c\u4ed6\u7684\u592a\u592a", "id": 3806605856656586}, {"user_id": 3896659137, "video_short_url": "http://t.cn/Rwvrtxs", "url": "http://weibo.com/3896659137/C2LQBakji", "timestamp": 1423049021000, "video_brief": "\u4fe9staff\u88ab\u6bbf\u4e0b\u8650\u89c6\u9891~~", "video_title": "\u3010\u4e1c\u4eac\u5965\u7279\u66fc\u6b63\u6708\u8282\u3011\u8d1d\u5229\u4e9a\u6bbf\u4e0b\u548c\u4e24\u4e2astaff", "content": "\u3010\u89c6\u9891\uff1a \u3010\u4e1c\u4eac\u5965\u7279\u66fc\u6b63\u6708\u8282\u3011\u8d1d\u5229\u4e9a\u6bbf\u4e0b\u548c\u4e24\u4e2astaff\u3011 \uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09 |\u3010\u4e1c\u4eac\u5965\u7279... ", "source": "\u4f18\u9177\u571f\u8c46", "comment_count": 0, "like_count": 1, "video_full_url": "http://v.youku.com/v_show/id_XODg2NDQ4NDg0.html", "user_href": "http://weibo.com/ultramanyy", "retweet_count": 12, "nickname": "\u5965\u7279\u66fc\u5c0f\u6c60\u5858", "id": 3806605852461356}, {"user_id": 1842161533, "video_short_url": "http://t.cn/R7CsGrv", "url": "http://weibo.com/1842161533/C2LPScDQl", "timestamp": 1423048913000, "video_brief": "\u5bcc\u58eb\u5eb7\u8d28\u68c0\u5458\u5f20\u5168\u86cb\u5927\u7206\u6599 \u82f9\u679c\u4e09\u661f\u5c0f\u7c73\u9524\u5b50\u5185\u5e55", "video_title": "\u5bcc\u58eb\u5eb7\u8d28\u68c0\u5458\u5f20\u5168\u86cb\u5927\u7206\u6599+\u82f9\u679c\u4e09\u661f\u5c0f\u7c73\u9524\u5b50\u5185\u5e55", "content": "\u5bcc\u58eb\u5eb7\u8d28\u68c0\u5458\u5f20\u5168\u86cb\u5927\u7206\u6599 \u82f9\u679c\u4e09\u661f\u5c0f\u7c73\u9524\u5b50\u5185\u5e55 |\u5bcc\u58eb\u5eb7\u8d28\u68c0...\uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09@\u4eba\u5728\u5f81\u9014yxb \u7b11\u5c4e\u6211\u4e86", "source": "\u5206\u4eab\u6309\u94ae", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XODE1MDk0OTY0.html", "user_href": "http://weibo.com/u/1842161533", "retweet_count": 0, "nickname": "\u6487\u6487\u5634\u8f6c\u8eab", "id": 3806605403013097}, {"user_id": 1795292387, "video_short_url": "http://t.cn/Rzs4mJl", "url": "http://weibo.com/1795292387/C2LPKyinu", "timestamp": 1423048895000, "video_brief": "2014\u5e74\u9a6c\u4e0a\u8981\u7ed3\u675f\u4e86\u3002\u8fd9\u4e00\u5e74\u4e86\u53d1\u751f\u4e86\u5f88\u591a\u5f88\u591a\u7684\u5927\u4e8b\u3002\u6709\u8ba9\u4eba\u6fc0\u52a8...", "video_title": "2014\u89c6\u9891\u5e74\u7ec8\u76d8\u70b9+\u56e0\u4f60\u800c\u4e0d\u5e73\u51e1", "content": "2014\u4e0d\u5e73\u51e1\u4e4b\u8def \u4f18\u9177\u5168\u89c6\u89d2\u5e74\u7ec8\u7279\u732e | |2014\u89c6\u9891\u5e74... \u4e0b\u8f7dAPP\uff1ahttp://t.cn/RvnJOPO (\u5206\u4eab\u81ea #\u9b54\u529b\u76d2# iPhone\u5ba2\u6237\u7aef)", "source": "\u9b54\u529b\u76d2", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XODUyMzAyNTUy.html?f=23218107&amp;amp;ev=1&amp;amp;from=y1.1-2.10001-0.1-1", "user_href": "http://weibo.com/u/1795292387", "retweet_count": 0, "nickname": "\u5c0f\u6b22\u6b22\u7231\u5403\u6c34\u679c", "id": 3806605328173800}, {"user_id": 2471402700, "video_short_url": "http://t.cn/zOJu0sC", "url": "http://weibo.com/2471402700/C2LPHgA1W", "timestamp": 1423048888000, "video_title": "\u51c0\u6c34\u5668\u989d\u9888\u9f99\u5934\u7684\u5b89\u88c5", "content": "\u51c0\u6c34\u5668\u989d\u9888\u9f99\u5934\u7684\u5b89\u88c5 \uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09 http://t.cn/zOJu0sC ", "source": "\u4f18\u9177\u571f\u8c46", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XMTY1MTkyMDky.html", "user_href": "http://weibo.com/u/2471402700", "retweet_count": 0, "nickname": "aishang\u9ad8\u8ddf\u7f8e\u817f", "id": 3806605293951752}, {"user_id": 1951762612, "video_short_url": "http://t.cn/RZLnY0h", "url": "http://weibo.com/1951762612/C2LPyqVhI", "timestamp": 1423048867000, "video_brief": "\u628a\u300a\u90e8\u843d\u51b2\u7a81\u300b\u7684\u6240\u6709\u5b98\u65b9\u5e7f\u544a\u5ba3\u4f20\u7247\u526a\u6210\u96c6\u9526\u4f1a\u662f\u4ec0\u4e48\u6837\u5462\uff1f\u4e00\u90e8...", "video_title": "\u300a\u90e8\u843d\u51b2\u7a81\u300b3D\u8ff7\u4f60\u7535\u5f71\u77ed\u7247", "content": "\u5b8c\u6574\u7248\u7684\u5728\u8fd9\u91cc \u300a\u90e8\u843d\u51b2\u7a81\u300b3D\u8ff7\u4f60\u7535\u5f71\u77ed\u7247 \uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09 |\u300a\u90e8\u843d\u51b2\u7a81... ", "source": "\u4f18\u9177\u571f\u8c46", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XODU5ODEzOTMy.html", "user_href": "http://weibo.com/u/1951762612", "retweet_count": 0, "nickname": "\u9648\u8000\u5ef7", "id": 3806605206416734}, {"user_id": 5222994527, "video_short_url": "http://t.cn/RwvBDQ8", "url": "http://weibo.com/5222994527/C2LPe1m6k", "timestamp": 1423048818000, "video_brief": "\u7a0d\u540e\u8865\u5145\u89c6\u9891\u7b80\u4ecb", "video_title": "\u5b59\u79d1\u821e\u8e48\u57f9\u8bad\u4e0a\u8bfe\u82b1\u7d6e|\u6210\u90fd\u53e4\u5178\u821e|\u53e4\u5178\u821e\u96c6\u8bad\u73ed", "content": "\u5b59\u79d1\u821e\u8e48\u57f9\u8bad\u4e0a\u8bfe\u82b1\u7d6e|\u6210\u90fd\u53e4\u5178\u821e|\u53e4\u5178\u821e\u96c6\u8bad\u73ed |\u5b59\u79d1\u821e\u8e48\u57f9...\uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09", "source": "\u5206\u4eab\u6309\u94ae", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XODg2MTg3NTMy.html", "user_href": "http://weibo.com/u/5222994527", "retweet_count": 0, "nickname": "\u51af\u591a\u591a618", "id": 3806605000323288}, {"user_id": 5228524809, "video_short_url": "http://t.cn/RZrgqG1", "url": "http://weibo.com/5228524809/C2LOQ6fbj", "timestamp": 1423048761000, "video_brief": "\u6ea7\u9633\u67d0\u8001\u677f\u513f\u5ab3\u5987\u627e\u9ed1\u793e\u4f1a\u6bb4\u6253\u4e00\u65e0\u624b\u65e0\u5bf8\u94c1\u7684\u5973\u5b50", "video_title": "\u624b\u673aQQ\u89c6\u9891_20150130164301", "content": "\u624b\u673aQQ\u89c6\u9891_20150130164301 \uff08\u5206\u4eab\u81ea @\u4f18\u9177. \u73b0\u5728\u793e\u4f1a\u98ce\u6c14\u600e\u4e48\u8fd9\u6837\uff0c\u5f53\u8857\u6bb4\u6253\u5973\u6027\u3002\u72d7\u5c41\u9ed1\u793e\u4f1a\uff0c\u6c5f\u82cf\u8fd9\u4e2a\u5730\u65b9\u592a\u5947\u602a\u4e86\u3002\u6c5f\u82cf\u4eba\u90fd\u8fd9\u6837\u561b\uff1f |\u624b\u673aQQ\u89c6\u9891... ", "source": "\u4f18\u9177\u571f\u8c46", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XODgyOTQ5MDE2.html", "user_href": "http://weibo.com/u/5228524809", "retweet_count": 0, "nickname": "\u8d75\u9e4f\u7a0b-dashboarder", "id": 3806604761488329}, {"user_id": 5359745777, "video_short_url": "http://t.cn/RZE6UIq", "url": "http://weibo.com/5359745777/C2LOxv7Jy", "timestamp": 1423048717000, "video_brief": "\u4e0d\u5fcd\u76f4\u89c6\uff01\u7b80\u76f4\u6b8b\u5fcd\uff014\u4e2a\u7238\u7238\u7ed9\u73a9\u513f\u574f\u4e86\u554a~\u5475\u5475\u54d2\uff01", "video_title": "\u9ec4\u78ca\u3001\u9646\u6bc5\u3001\u6768\u5a01\u3001\u66f9\u683c\u6700\u60f3\u5220\u6389\u7684\u89c6\u9891~~\u4e48\u4e48\u54d2\uff01", "content": "\u9ec4\u78ca\u3001\u9646\u6bc5\u3001\u6768\u5a01\u3001\u66f9\u683c\u6700\u60f3\u5220\u6389\u7684\u89c6\u9891~~\u4e48\u4e48\u54d2\uff01 |\u9ec4\u78ca\u3001\u9646\u6bc5...\uff08\u5206\u4eab\u81ea @\u4f18\u9177\uff09", "source": "\u5206\u4eab\u6309\u94ae", "comment_count": 0, "like_count": 0, "video_full_url": "http://v.youku.com/v_show/id_XODc5ODMzMzM2.html", "user_href": "http://weibo.com/u/5359745777", "retweet_count": 0, "nickname": "\u666e\u901a\u4ebaVS\u571f\u8c6a", "id": 3806604577417900}]}